spec/models/user_spec.rb

```diff
@@ -94,6 +94,16 @@
       expect(user).to be_valid
       expect(user.login).to eq "foobar"
     end
+
+    describe "uniqueness" do
+      before { @user = create(:user, login: "jayita") }
```

Comment:
We generally don't use ivars in rspec tests. It's kind of a code smell because `let` is a better way to accomplish this. So you can do `let(:user) { create(...) }`.